bundle/DependencyInjection/Compiler/FieldTypeFormMapperDispatcherPass.php

```diff
@@ -40,16 +46,36 @@ public function process(ContainerBuilder $container)
     }
 
     /**
-     * Gathers services tagged as either ez.fieldFormMapper.value or ez.fieldFormMapper.definition.
+     * Gathers services tagged as either
+     * - ez.fieldFormMapper.value (deprecated)
+     * - ez.fieldFormMapper.definition (deprecated)
+     * - ezplatform.field_type.form_mapper.value
+     * - ezplatform.field_type.form_mapper.definition.
      *
      * @param \Symfony\Component\DependencyInjection\ContainerBuilder $container
      *
      * @return array
      */
-    private function findTaggedFormMapperServices(ContainerBuilder $container)
+    private function findTaggedFormMapperServices(ContainerBuilder $container): array
     {
-        return
-            $container->findTaggedServiceIds('ez.fieldFormMapper.value') +
-            $container->findTaggedServiceIds('ez.fieldFormMapper.definition');
+        $ezFieldFormMapperValueTags = $container->findTaggedServiceIds(self::FIELD_FORM_MAPPER_VALUE);
+        $ezFieldFormMapperDefinitionTags = $container->findTaggedServiceIds(self::FIELD_FORM_MAPPER_DEFINITION);
+        $ezplatformFieldFormMapperValueTags = $container->findTaggedServiceIds(self::EZPLATFORM_FIELD_TYPE_FORM_MAPPER_VALUE);
+        $ezplatformFieldFormMapperDefinitionTags = $container->findTaggedServiceIds(self::EZPLATFORM_FIELD_TYPE_FORM_MAPPER_DEFINITION);
+
+        foreach ($ezFieldFormMapperValueTags as $ezFieldFormMapperValueTag) {
+            @trigger_error('`ez.fieldFormMapper.value` service tag is deprecated and will be removed in version 9. Please use `ezplatform.field_type.form_mapper.value` instead.', E_USER_DEPRECATED);
```

Comment:
In version 9? That's too long :)
As this lands in container log, maybe let's use product name: eZ Platform 4.0.
CS: Please reformat it into multiple lines so it's easier to review, as you can see 2/3 of reviewers missed that. Since you have constant for that tag, you could use `sprintf` for instance.